compiler/src/iree/compiler/Codegen/Common/TransformExtensions/TransformMatchers.cpp

```diff
@@ -152,6 +228,42 @@ transform_dialect::StructuredOpMatcher::output(int64_t position,
   return *this;
 }
 
+transform_dialect::StructuredOpMatcher &
+transform_dialect::StructuredOpMatcher::output(int64_t position,
```

Comment:
Skipping this in a first reading, let's discuss live, looks interesting!

Reply:
Discussed offline, nice!
Looking forward to see how this will also improve the C++ APIs in a followup PR.